src/cpack.ts

```diff
@@ -89,13 +89,6 @@ export class CPackDriver implements vscode.Disposable {
             }
         }
 
-        const opts = driver.expansionOptions;
-        const args = [];
-        for (const value of this.ws.config.cpackArgs) {
```

Comment:
If this is doubling, why don't we keep this section and remove the corresponding if statements?

Reply:
Good question. I figured either of the two places is a good candidate to stay or be removed. One place in here (getCPackArgs) and one in runCPack. The reason that made me chose this one would be that these args are not coming from the preset but from settings, like one additional thing defined by the users. The other switches constructed during getCPackArgs are coming from fields defined in the preset. Thank you for the review!